bin/cli.js

```diff
@@ -483,6 +484,53 @@ program
     }
   });
 
+// Development mode
+program
+  .command("development-mode")
+  .description("Development mode - Watch for changes in files")
+  .requiredOption(
+    "-f, --firm <firm-id>",
+    "Specify the firm to be used",
+    firmIdDefault
+  )
+  .option(
+    "-h, --handle <handle>",
+    "Watch for changes in liquid and yaml files related to the reconcilation mentioned. Run a new Liquid Test on each save"
+  )
+  .option(
+    "-u, --update-templates",
+    "Watch for changes in any liquid file. Publish the new code of the template into the Platform on each save"
+  )
+  .option(
+    "-t, --test <test-name>",
+    "Specify the name of the test to be run (optional). It has to be used together with --handle",
+    ""
+  )
+  .option(
+    "--html",
+    "Get a html file of the template generated with the Liquid Test information (optional). It has to be used together with --handle",
+    false
+  )
+  .option("--yes", "Skip the prompt confirmation (optional)")
```

Comment:
I think the `--yes` option is not needed here

Reply:
I implemented considering the flag `--update-templates`. Similar check to when you use update-reconciliation` or `update-shared-part` commands (you are making changing to the liquid code in the Platform, in the environment you defined